packages/dialog/src/__tests__/Dialog.spec.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react'
+import { render, fireEvent } from '@testing-library/react'
+import { Dialog, DialogContent, DialogFooter } from '../'
+import { Text, Button } from '@kodiak-ui/primitives'
+
+function DialogExample() {
+  const [isOpen, setIsOpen] = React.useState(false)
+
+  return (
+    <>
+      <div data-testid="outside">
+        Some content outside to click on to close the modal
+      </div>
+      <Button onClick={() => setIsOpen(true)}>Show default example</Button>
+      <Dialog
+        isOpen={isOpen}
+        title={
+          <div>
+            <Text as="h3" mb={2}>
+              Testing
+            </Text>
+            <Text mb={0}>Testing some text</Text>
+          </div>
+        }
+        // eslint-disable-next-line @typescript-eslint/no-empty-function
+        onDismiss={() => setIsOpen(false)}
+        aria-label="Warning about next steps"
+      >
+        <DialogContent>
+          <Text as="p">
+            Contrary to popular belief, Lorem Ipsum is not simply random text.
+            It has roots in a piece of classical Latin literature from 45 BC,
+            making it over 2000 years old. Richard McClintock, a Latin professor
+            at Hampden-Sydney College in Virginia, looked up one of the more
+            obscure Latin words, consectetur, from a Lorem Ipsum passage, and
+            going through the cites of the word in classical literature,
+            discovered the undoubtable source. Lorem Ipsum comes from sections
+            1.10.32 and 1.10.33 of &quot;de Finibus Bonorum et Malorum&quot;
+            (The Extremes of Good and Evil) by Cicero, written in 45 BC. This
+            book is a treatise on the theory of ethics, very popular during the
+            Renaissance. The first line of Lorem Ipsum, &quot;Lorem ipsum dolor
+            sit amet..&quot;, comes from a line in section 1.10.32. The standard
+            chunk of Lorem Ipsum used since the 1500s is reproduced below for
+            those interested. Sections 1.10.32 and 1.10.33 from &quot;de Finibus
+            Bonorum et Malorum&quot; by Cicero are also reproduced in their
+            exact original form, accompanied by English versions from the 1914
+            translation by H. Rackham.
+          </Text>
+        </DialogContent>
+        <DialogFooter>
+          {/* eslint-disable-next-line @typescript-eslint/no-empty-function */}
+          <Button
+            ml="auto"
```

Comment:
Just a thought, I like having these mini props for quick styling, but I wonder if when we start to have a group of them on complex (ie. not layout) components that we put everything into the `sx` prop because it helps split out the difference between presentational properties and actual props that control the component.

Reply:
Yea, that could be a good idea. Let's see how it all works out when we try to use these components to build the templates.